macros/ArticlesByTag.ejs

```diff
@@ -66,9 +66,9 @@ URL_ROOTS.forEach(function (root) {
 
         if (match) {
             ARTICLES.push({
-                title   : p.title,
+                title   : kuma.htmlEscape(p.title),
                 url     : p.url,
-                summary : p.summary
+                summary : mdn.escapeQuotes(p.summary)
```

Comment:
I suspect ``<%=`` should be used as well (line 90), rather than manually escaping quotes. Wouldn't HTML be just as bad in the summary section, and easier to do for a malicious user?